wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/MecanumControllerCommand.java

```diff
@@ -112,9 +112,9 @@ public MecanumControllerCommand(
 
     m_controller =
         new HolonomicDriveController(
-            requireNonNullParam(xController, "xController", "SwerveControllerCommand"),
-            requireNonNullParam(yController, "xController", "SwerveControllerCommand"),
-            requireNonNullParam(thetaController, "thetaController", "SwerveControllerCommand"));
+            requireNonNullParam(xController, "xController", "MecanumControllerCommand"),
+            requireNonNullParam(yController, "xController", "MecanumControllerCommand"),
```

Comment:
There's another typo here:
```suggestion
            requireNonNullParam(yController, "yController", "MecanumControllerCommand"),
```